shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -528,15 +551,16 @@ - (void)testHideOverlay {
 }
 
 - (void)testNotifyLowMemory {
-  FlutterViewController* viewController =
-      [[FlutterViewController alloc] initWithEngine:self.mockEngine nibName:nil bundle:nil];
-  OCMStub([self.mockEngine viewController]).andReturn(viewController);
+  FlutterEnginePartialMock* mockEngine = [[FlutterEnginePartialMock alloc] init];
+  FlutterViewController* viewController = [[FlutterViewController alloc] initWithEngine:mockEngine
+                                                                                nibName:nil
+                                                                                 bundle:nil];
   id viewControllerMock = OCMPartialMock(viewController);
   OCMStub([viewControllerMock surfaceUpdated:NO]);
-
   [viewController beginAppearanceTransition:NO animated:NO];
   [viewController endAppearanceTransition];
-  OCMVerify([self.mockEngine notifyLowMemory]);
+  XCTAssertTrue(mockEngine.didCallNotifyLowMemory);
```

Comment:
If you can actually allocate a `FlutterEngine` subclass without mocking is the subclass necessary?  Does it work if you want for an `XCTKVOExpectation`?
```obj-c
[self keyValueObservingExpectation:mockEngine keyPath:@"notifyLowMemory" handler:nil]
```

Reply:
You mean could we use a real FlutterEngine?  There shouldn't be a keyPath `notifyLowMemory` on a standard FlutterEngine, that would only get created if there was a property called `notifyLowMemory` or if we manually registered one.

Reply:
(the subclass is also necessary for setting the lifecycle channel, not just needed for the notifyLowMemory stuff)